egui_demo_app/src/lib.rs

```diff
@@ -19,23 +28,48 @@ pub(crate) fn seconds_since_midnight() -> f64 {
 #[cfg(target_arch = "wasm32")]
 use eframe::wasm_bindgen::{self, prelude::*};
 
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen]
+pub struct WebHandle {
+    handle: Arc<Mutex<AppRunner>>,
+}
```

Comment:
This `WenHandle` wrapper should be moved into `eframe` instead so that all `eframe` users can stop their instances!

Reply:
Again, I tried to make egui embeddable, not just run in whole document. WebHandle is extensible. You have handle in js and you can call some methods to interact with egui app

Reply:
Sure, but what is the point of defining `WebHandle` in the example app? I don't see `stop_web()` being used anywhere. Is it dead code? Should `docs/index.html` store the handle as a demonstration?

Reply:
Well. Ok. I'll create another wrapper in eframe